core/src/commonMain/kotlin/BottomSheet.kt

```diff
@@ -422,6 +424,8 @@ fun BottomSheet(
                             val requireOffset = state.anchoredDraggableState.requireOffset()
                             val y = requireOffset.toInt()
                             IntOffset(x = 0, y = y)
+                        } else if(containerHeight == Dp.Unspecified) {
```

Comment:
This doesn't seem to be formatted - i think if is missing a space. Please format the code

Reply:
Fixed!